README.md

```diff
@@ -81,7 +81,7 @@ You can find more examples with different color schemes in [Screenshots](https:/
 For correct work you will first need:
 
 * [`zsh`](http://www.zsh.org/) (v5.2 or recent) must be installed.
-* [Powerline Font](https://github.com/powerline/fonts) must be installed and used in your terminal.
+* [Powerline Font](https://github.com/powerline/fonts) must be installed and used in your terminal (for example, switch font to "Fira Mono for Powerline")
```

Comment:
If you want to give an example, I'd recommend showcasing the [Fira Code](https://github.com/tonsky/FiraCode) instead — it seems to be a fairly popular font (I use it too), it has ligatures and it has powerline symbols that are in style with the font itself.

Reply:
I would prefer Fira Code too. Please change.